### PR TITLE
blueprints: Add initial blueprints

### PR DESCRIPTION
**Pull request by luise**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FROM node:8
+MAINTAINER Luise Valentin <luise@quilt.io>
+
+RUN npm install -g slackin@0.13.0
```
Comment (kklin) [inline Dockerfile]: So simple!

I've gone ahead and setup autobuild on Dockerhub.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
 # Slackin for Quilt
 This repository implements a [Slackin](https://github.com/rauchg/slackin) specification
 for [Quilt](http://quilt.io).
+
+To create a new Slackin service, simply pass your Slack team ID and Slack token to the
+constructor:
+
+```javascript
+var slackin = Slackin.New("YOUR_SLACK_ID", "YOUR_SLACK_TOKEN");
```
Comment (kklin) [inline README.md]: I don't like the `Slackin` is uppercase -- that's usually reserved for objects.

File: main.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+const { createDeployment, Machine } = require("@quilt/quilt");
+const Slackin = require("./slackin");
```
Comment (kklin) [inline main.js]: Same here re capitalization

File: main.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+// From the domain of your login page (i.e. https://YOUR_SLACK_ID.slack.com).
```
Comment (kklin) [inline main.js]: I think "slack team" is the term that slack uses, not sure though.

File: main.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+var slack_id = "YOUR_SLACK_ID";
```
Comment (kklin) [inline main.js]: Also, might as well make these consts as well

File: main.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+var slackin = Slackin.New(slack_id, slack_token);
+
+var deployment = createDeployment();
+var machine = new Machine({provider: "Amazon", size: "t2.micro"});
+
+deployment.deploy(machine.asMaster());
+deployment.deploy(machine.asWorker());
+deployment.deploy(slackin);
+
```
Comment (kklin) [inline main.js]: Extra newline

File: slackin.js [hunk 7]
```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+var { Container, Service, publicInternet } = require("@quilt/quilt");
+
+exports.New = function(slack_team_id, slack_token, port = 80) {
```
Comment (kklin) [inline slackin.js]: Default arguments! ✨

File: slackin.js [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+		}
+	);
+
+	var slackin = new Service("slackin", [container]);
+	slackin.allowFrom(publicInternet, port);
+	publicInternet.allowFrom(slackin, 443);
+
+	return slackin;
+};
+
```
Comment (kklin) [inline slackin.js]: Extra newline

File: main.js (hunk 4 above)
Comment (luise) [inline main.js]: Hm, seems like you can actually change your team name and URL separately. I could call it "Team ID" or "Slack Subdomain". Do you have a preference, @kklin ?

**Comment by luise** [comment]
@kklin , addressed your comments!

**Pull request merged**